www/app.py

```diff
@@ -2,11 +2,58 @@
 from os import listdir
 from os.path import isfile, join
 import xml.etree.ElementTree
+from flask_socketio import SocketIO, emit
 import Adafruit_GPIO.PWM as pwmLib
+import Adafruit_GPIO.GPIO as gpioLib
 
+# Let SocketIO choose the best async mode
+async_mode = None
 app = Flask(__name__)
+socketio = SocketIO(app, async_mode=async_mode)
+thread = None
 
 pwm = pwmLib.get_platform_pwm(pwmtype="softpwm")
+gpio = gpioLib.get_platform_gpio();
+
+binary_sensors = []
+
+class gpioPoller:
```

Comment:
Since this is no longer a thread and is just a container, I don't think the name `gpioPoller `makes much sense.